<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysicalDeviceWidget</class>
 <widget class="QWidget" name="PhysicalDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Properties</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Features</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Limits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Memory</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Formats</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QLineEdit" name="driverVersion"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="vendorId"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Pipeline Cache
UUID</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Device ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="type">
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Integrated</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Discrete</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virtual GPU</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CPU</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="apiVersion"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Vendor ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>API Version</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="deviceId"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Driver Version</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="uuid"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
     <widget class="QWidget" name="page_3"/>
     <widget class="QWidget" name="page_4"/>
     <widget class="QWidget" name="page_5"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
